src/pages/[platform]/build-a-backend/restapi/existing-resources/index.mdx

```diff
@@ -43,11 +43,17 @@ import flutter2 from '/src/fragments/lib/restapi/existing-resources.mdx';
 <Fragments fragments={{ flutter: flutter2 }} />
 
 <InlineFilter filters={['javascript','angular','nextjs','react','vue','react-native']}>
-Existing Amazon API Gateway resources can be used with the Amplify Libraries by calling `Amplify.configure()` with the API Gateway API name and options. Note, you will need to supply the full resource configuration and library options objects when calling `Amplify.configure()`. The following example shows how to configure additional API Gateway resources to an existing Amplify application:
 
-```javascript
+Existing Amazon API Gateway resources can be used with the Amplify Libraries by calling `Amplify.configure()` with the API Gateway API name and options. Note, you need to supply the full resource configuration and library options objects when calling `Amplify.configure()`. The following example shows how to configure additional API Gateway resources to an existing Amplify application:
+
+```typescript
+import { Amplify } from 'aws-amplify';
+import amplifyconfig from './src/amplifyconfiguration.json';
+Amplify.configure(amplifyconfig):
+
 const existingConfig = Amplify.getConfig();
 
+// Add existing resource to the existing configuration.
```

Comment:
nit: We can add backend / AWS / infrastructure resources
```suggestion
// Add existing resources to the existing configuration.
```

Reply:
Comment is for this example where we are adding only one resource 😊